javascriptv3/example_code/ssm/hello.js

```diff
@@ -0,0 +1,32 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[ssm.JavaScript.Basics.hello]
+import { paginateListDocuments, SSMClient } from "@aws-sdk/client-ssm";
+
+// Call ListDocuments and display the result.
+export const main = async () => {
+  const client = new SSMClient();
+  let listDocumentsPaginated = [];
+  console.log(
+    "Hello, AWS Systems Manager! Let's list some of your documents:\n"
+  );
+  try {
+    // The paginate function is a wrapper around the base command.
+    const paginator = paginateListDocuments({ client }, { MaxResults: 5 });
+    for await (const page of paginator) {
+      listDocumentsPaginated.push(...page.DocumentIdentifiers);
+    }
+  } catch (caught) {
+    console.error(`There was a problem saying hello: ${caught.message}`);
+    throw caught;
+  }
+  console.log(listDocumentsPaginated);
```

Comment:
```
Hello, AWS Systems Manager! Let's list some of your documents:

[
  {
    CreatedDate: 2018-02-15T02:29:52.370Z,
    DocumentFormat: 'JSON',
    DocumentType: 'Automation',
    DocumentVersion: '1',
...
  },
  {
    CreatedDate: 2018-02-15T02:29:58.115Z,
    DocumentFormat: 'JSON',
    DocumentType: 'Automation',
    DocumentVersion: '1',
...
  },
```
(the dots represent more data fields)

Reply:
Personally I'd reduce the amount of info shown and do something like:
`• ${Name} - ${DocumentFormat} - ${CreatedDate}`
It makes it more readable and provides a peek in to the response instead of showing the whole thing. I'm fine with this though.